README.md

```diff
@@ -1,4 +1,4 @@
-# Docker Official Images
+# Docker Official Images 
```

Comment:
??? :confused:

Reply:
That is strange.  I will remove that extra space.